src/linear_algebra/dimension.lean

```diff
@@ -231,9 +234,9 @@ theorem dim_quotient_le (p : submodule R M) :
 
 variables [nontrivial R]
 
-lemma {m} cardinal_lift_le_dim_of_linear_independent
+lemma cardinal_lift_le_dim_of_linear_independent
   {ι : Type w} {v : ι → M} (hv : linear_independent R v) :
-  cardinal.lift.{max v m} (#ι) ≤ cardinal.lift.{max w m} (module.rank R M) :=
+  cardinal.lift.{v} (#ι) ≤ cardinal.lift.{w} (module.rank R M) :=
```

Comment:
Perhaps I'm missing something, but isn't this less general now?

Reply:
That's the whole point of this PR.